defaults/main.yml

```
@@ -38,5 +38,8 @@ userli_munin: false
 userli_primary: true
 userli_webserver: apache2
 
+# Set default language for navbar
+userli_services_locale: "en"
```

```suggestion
userli_parameters_locale: "en"
```

Would reflect the name from the config for this property.